packages/test-utils/src/legacy-cli/client.ts

```diff
@@ -84,7 +83,8 @@ class ProtocolClient {
     callback?: (err?: Error) => void
   ) {
     const id = this.nextMessageId++;
-    debug("Sending command %s: %o", method, { id, params, sessionId });
+    logger.info("SendCommand:Started", { id, params, sessionId });
```

Comment:
Not sure how much we care about this, but since we explicitly mask the access token in a few places– this call site _could_ contain that token (or possibly other sensitive info?) e.g. we send the access token to this when we call the `Authentication.setAccessToken` API.

Maybe our `formatTags` helper could check for a few keys by name (like `apiKey`, `accessToken`, or `token`) and make sure they're masked?